test/scripts/e2e_subs/e2e-app-simulate.sh

```diff
@@ -106,7 +106,7 @@ fi
 # WE ALSO TEST OVERSPEND IN TRANSACTION GROUP #
 ###############################################
 
-${gcmd} clerk send -a 1000000000000 -f ${ACCOUNT} -t ${ACCOUNT} -o "${TEMPDIR}/pay1.tx"
+${gcmd} clerk send -a 1000000000000000 -f ${ACCOUNT} -t ${ACCOUNT} -o "${TEMPDIR}/pay1.tx"
```

Comment:
There was a flaky test run where this did not overspend, so I increased the amount